### ISSUE TITLE
SqueezeNet Pruning

### ISSUE THREAD

**Issue by Kuldeep-Attri** [jacobgil/pytorch-pruning#14]
Has anyone tried pruning the **SqueezeNet** using this method and the program?
I have been trying to prune squeezenet but the test accuracy, during the finetuning after pruning first set of filter, is always 0.5, any idea what might be wrong?

I am confused about which filter to remove after getting the 'filter_index' from the 'compute_rank()' method.

Thank you!!!

**Comment by Kuldeep-Attri** [comment]
Modified program for pruning **SqueezeNet**:
https://github.com/Kuldeep-Attri/Pruning-CNN
